docs/strict-mode.md

```diff
@@ -310,3 +311,8 @@ Both "properties" and "required" need `type: "object"` to satisfy `strictTypes`
 ### Strict number validation
 
 Strict mode also affects number validation. By default Ajv fails `{"type": "number"}` (or `"integer"`) validation for `Infinity` and `NaN`.
+
+
+### Strict required
+
+An additional option `strictRequired` ("log" by default) validates that the `required` keyword contains only keys that are explicitly defined in the `properties` key of the object they are referencing.
```

Comment:
Suggest rewording as other sections - as is it may be read as opt-in behaviour, rather than opt-out.